openapi.yaml

```diff
@@ -957,6 +957,50 @@ paths:
       - lang: CLI
         source: >
           linode-cli account login-view 1234
+  /account/maintenance:
```

Comment:
This endpoint is in beta for the moment, because we're not entirely sure if the returned data schema won't need to change once cloud starts working on their end.

Reply:
added beta tag and beta to the curl sample url